google-cloud-firestore/clirr-ignored-differences.xml

```diff
@@ -223,6 +223,30 @@
     <to>java.util.List</to>
   </difference>
 
+  <!--
+  UpdateBuilder
+  -->
+  <difference>
+    <differenceType>6001</differenceType>
+    <className>com/google/cloud/firestore/UpdateBuilder</className>
+    <field>pendingOperations</field>
+  </difference>
+  <difference>
+    <differenceType>6001</differenceType>
+    <className>com/google/cloud/firestore/UpdateBuilder</className>
+    <field>state</field>
+  </difference>
+  <difference>
+    <differenceType>6010</differenceType>
+    <className>com/google/cloud/firestore/UpdateBuilder</className>
+    <field>writes</field>
+  </difference>
+  <difference>
+    <differenceType>7002</differenceType>
+    <className>com/google/cloud/firestore/UpdateBuilder</className>
+    <method>int getMutationsSize()</method>
```

Comment:
We can't remove this method without a major version release since it's public. because it's public customers could be using already and we won't want to remove it and break them.

We should be able to keep the method public without any issues.